src/react-components/codebox.tsx

```diff
@@ -19,18 +19,19 @@ export const CodeBox = ({
   className,
   ...props
 }: CodeBoxProps): JSX.Element => {
+  const id = Math.random().toString(36).substring(2)
```

Comment:
This is regenerated on each render. Is that really what we want here? Why does it need to be random?

Reply:
because we can have multiple codeboxes on a singular page. If the IDs are all the same you will collapse and expand all of them when clicking the "hide content checkbox"

Reply:
Sure, but maybe you could memoize the value instead, so it's not regenerated on _each_ render?

An explicit `id` prop would be even better, though.

Reply:
Alright that makes sense, I'm just not sure what impact that will have on the express (pure html) support.

Reply:
Yeah it breaks the express app.